drkafka/src/main/java/com/pinterest/doctorkafka/security/ExampleAuthorizationFilter.java

```diff
@@ -0,0 +1,66 @@
+package com.pinterest.doctorkafka.security;
```

Comment:
The name `ExampleAuthorizationFilter` seems awkward to me. How about name it as `AuthorizationFilterDemo`?

Reply:
Renamed to SampleAuthorizationFilter